xray/src/bin/merge_xray_quadtrees.rs

```diff
@@ -77,12 +76,18 @@ fn read_metadata_from_directories(directories: &[PathBuf]) -> Vec<Meta> {
         .collect()
 }
 
-fn get_root_node(meta: &Meta) -> Option<NodeId> {
-    meta.nodes.iter().copied().min_by_key(|node| node.level())
+fn get_root_node(meta: &Meta) -> Option<Node> {
```

Comment:
Let's make this function a public method of`Meta`?

Reply:
Done

Reply:
:+1: